app/uk/gov/hmrc/helptosavestub/controllers/PayePersonalDetailsController.scala

```diff
@@ -70,6 +71,12 @@ class PayePersonalDetailsController extends BaseController with DESController wi
     date ← Gen.choose(100L, 1000L).map(LocalDate.ofEpochDay)
     address ← Gen.ukAddress
     postcode ← Gen.postcode
+    countryCode ← Gen.choose(1, 250)
+    callingCode ← Gen.choose(1, 250)
+    telephoneType ← Gen.choose(1, 7)
```

Comment:
in order to increase the chances of getting a phone number to use we could do something like `Gen.oneOf(1,2,7)` so that there a 2 in 3 chance of the phone number being populated instead of 2 in 7